4.0/en/0x15-V8-Error-Logging.md

```diff
@@ -28,6 +32,14 @@ If logs contain private or sensitive data, the definition of which varies from c
 | **8.10** | Verify that all authentication decisions are logged, without storing sensitive session identifiers or memorized secrets. This should include requests with relevant metadata needed for security investigations.  | ✓ | ✓ | ✓ | 4.0 |
 | **8.11** | Verify that a common logging format and approach is used across the system.  | ✓ | ✓ | ✓ | 4.0 |
 
+### 8.2 Error Handling
+
+| # | Description | L1 | L2 | L3 | Since |
+| --- | --- | --- | --- | -- | -- |
+| **8.2.1** | Verify that a generic message is shown when an error occurs, potentially with a unique ID which support personnel can use to investigate.  | ✓ | ✓ | ✓ | 4.0 |
+| **8.3.2** | Verify that exception handling is used across the codebase to account for expected or unexpected error conditions.  | ✓ | ✓ | ✓ | 4.0 |
```

Comment:
Shouldn't this be expected AND unexpected error conditions?
Otherwise this could be understood to either handle all known exceptions (i.e. catch only exceptions you throw yourself) or to have a fallback exception handling (i.e. a generic exception handler) and skip properly handling known exceptions intentionally thrown by application code.

Reply:
Basically that's what 8.3.3 is for, but I still feel like this rule is easy to be misunderstood.

Reply:
Fair one, updated.

Reply:
Just noticed: You go from 8.2.1 to 8.3.2 and 8.3.3.
Also - but this might be on your roadmap anyways - you added a 3rd numbering component on 8.2, but haven't updated the 8.1 entries to use a middle component as well.

Reply:
Yeah numbering is on the list of things to do at the end :)